tools/ui-components/tailwind.config.js

```diff
@@ -91,5 +104,9 @@ module.exports = {
       }
     }
   },
-  plugins: []
+  plugins: [
+    plugin(({ addVariant }) => {
+      addVariant('aria-disabled', '[aria-disabled="true"]');
+    })
+  ]
```

Comment:
Tailwind supports a lot of selectors by default ([ref](https://tailwindcss.com/docs/hover-focus-and-other-states#quick-reference)), but `aria-disabled` isn't one of them. I followed [this guide](https://tailwindcss.com/docs/hover-focus-and-other-states#creating-custom-modifiers) to add our own modifier. This allows us to style the disabled state using `aria-disabled:...` classes.

And at this point, I have to say I'm very impressed with Tailwind. They support everything!!! And their documentation is very well-written.